updates_log.info.yml

```diff
@@ -4,6 +4,6 @@ description: "Log module update info"
 core: 8.x
 core_version_requirement: ^8 || ^9 || ^10
 package: Development
-version: 2.0.3
+version: 2.2.0
```

Comment:
I think we should use 2.1.0 as the other commit as these would go Together

Reply:
Merged `main`, and used the `2.1.0`.
Thanks for the idea!